big_tests/tests/graphql_muc_light_SUITE.erl

```diff
@@ -537,6 +580,229 @@ user_blocking_list_story(Config, Alice, Bob) ->
+    RoomName = <<"first room">>,
+    MsgBody = <<"Hello there!">>,
+    {ok, #{jid := RoomJID}} = create_room(MUCServer, RoomName, <<"subject">>, AliceBisBin),
+    {ok, _} = invite_user(RoomJID, AliceBisBin, BobBin),
+    % Try with a non-existent domain
+    Res = send_message_to_room(jid:to_binary(RoomJID), InvalidUser, MsgBody, Config),
+    get_unauthorized(Res),
+    % Try with an external domain room
+    Res2 = send_message_to_room(jid:to_binary(RoomJID), AliceBisBin, MsgBody, Config),
+    get_unauthorized(Res2).
+
+domain_admin_get_room_messages_no_permission(Config) ->
+    escalus:fresh_story_with_config(Config, [{alice, 1}],
+                                    fun domain_admin_get_room_messages_no_permission_story/2).
+
+domain_admin_get_room_messages_no_permission_story(Config, Alice) ->
+    AliceBin = escalus_client:short_jid(Alice),
+    MUCServer = ?config(muc_light_host, Config),
+    RoomName = <<"first room">>,
+    ExternalDomain = domain_helper:secondary_domain(),
+    Limit = 40,
+    {ok, #{jid := #jid{luser = RoomID}}} =
+    create_room(MUCServer, RoomName, <<"subject">>, AliceBin),
+    % Try with a non-existent domain
+    Res = get_room_messages(make_bare_jid(RoomID, ?UNKNOWN_DOMAIN), Limit, null, Config),
+    get_unauthorized(Res),
+    % Try with an external domain
+    Res2 = get_room_messages(make_bare_jid(RoomID, ExternalDomain), Limit, null, Config),
+    get_unauthorized(Res2).
+
+domain_admin_list_user_rooms_no_permission(Config) ->
+    escalus:fresh_story_with_config(Config, [{alice, 1}],
+                                    fun domain_admin_list_user_rooms_no_permission_story/2).
+
+domain_admin_list_user_rooms_no_permission_story(Config, Alice) ->
+    AliceBin = escalus_client:short_jid(Alice),
+    MUCServer = ?config(muc_light_host, Config),
+    RoomName = <<"first room">>,
+    ExternalDomain = domain_helper:secondary_domain(),
+    {ok, #{jid := #jid{luser = RoomID}}} =
+    create_room(MUCServer, RoomName, <<"subject">>, AliceBin),
+    % Try with a non-existent domain
+    Res = list_user_rooms(make_bare_jid(RoomID, ?UNKNOWN_DOMAIN), Config),
+    get_unauthorized(Res),
+    % Try with an external domain
+    Res2 = list_user_rooms(make_bare_jid(RoomID, ExternalDomain), Config),
+    get_unauthorized(Res2).
+
+domain_admin_list_room_users_no_permission(Config) ->
+    escalus:fresh_story_with_config(Config, [{alice, 1}],
+                                    fun domain_admin_list_room_users_story_no_permission/2).
+
+domain_admin_list_room_users_story_no_permission(Config, Alice) ->
+    AliceBin = escalus_client:short_jid(Alice),
+    MUCServer = ?config(muc_light_host, Config),
+    RoomName = <<"first room">>,
+    ExternalDomain = domain_helper:secondary_domain(),
+    {ok, #{jid := #jid{luser = RoomID}}} = create_room(MUCServer, RoomName, <<"subject">>, AliceBin),
+    % Try with a non-existent domain
+    Res = list_room_users(make_bare_jid(RoomID, ?UNKNOWN_DOMAIN), Config),
```

Comment:
I have to note that both jid's in this test point to non-existing rooms at domains that are not MUC-Light domains. It would be better to test an existing room at a different domain.